feature/security/gnsi/certz/tests/server_certificates/server_certificates_test.go

```diff
@@ -0,0 +1,240 @@
+			clientCertFile:  dirPath + "ca-10/client-ecdsa-a-cert.pem",
+			clientKeyFile:   dirPath + "ca-10/client-ecdsa-a-key.pem",
+		},
+		{
+			desc:            "Certz2.1:Load server certificate of rsa keytype with 1000CA configuration",
+			serverCertFile:  dirPath + "ca-1000/server-rsa-a-cert.pem",
+			serverKeyFile:   dirPath + "ca-1000/server-rsa-a-key.pem",
+			trustBundleFile: dirPath + "ca-1000/trust_bundle_1000_rsa.pem",
+			clientCertFile:  dirPath + "ca-1000/client-rsa-a-cert.pem",
+			clientKeyFile:   dirPath + "ca-1000/client-rsa-a-key.pem",
+		},
+		{
+			desc:            "Certz2.1:Load server certificate of ecdsa keytype with 1000CA configuration",
+			serverCertFile:  dirPath + "ca-1000/server-ecdsa-a-cert.pem",
+			serverKeyFile:   dirPath + "ca-1000/server-ecdsa-a-key.pem",
+			trustBundleFile: dirPath + "ca-1000/trust_bundle_1000_ecdsa.pem",
+			clientCertFile:  dirPath + "ca-1000/client-ecdsa-a-cert.pem",
+			clientKeyFile:   dirPath + "ca-1000/client-ecdsa-a-key.pem",
+		},
+	}
+
+	for _, tc := range cases {
+		t.Run(tc.desc, func(t *testing.T) {
+
+			san := setupService.ReadDecodeServerCertificate(t, tc.serverCertFile)
+			serverCert := setupService.CreateCertzChain(t, setupService.CertificateChainRequest{
+				RequestType:    setupService.EntityTypeCertificateChain,
+				ServerCertFile: tc.serverCertFile,
+				ServerKeyFile:  tc.serverKeyFile})
+			serverCertEntity := setupService.CreateCertzEntity(t, setupService.EntityTypeCertificateChain, &serverCert, "servercert")
+
+			trustCertChain := setupService.CreateCertChainFromTrustBundle(tc.trustBundleFile)
+			trustBundleEntity := setupService.CreateCertzEntity(t, setupService.EntityTypeTrustBundle, trustCertChain, "cabundle")
+
+			cert, err := tls.LoadX509KeyPair(tc.clientCertFile, tc.clientKeyFile)
+			if err != nil {
+				t.Fatalf("Failed to load  client cert: %v", err)
+			}
+
+			cacert := x509.NewCertPool()
+			cacertBytes, err := os.ReadFile(tc.trustBundleFile)
+			if err != nil {
+				t.Fatalf("Failed to read ca bundle :%v", err)
+			}
+			if ok := cacert.AppendCertsFromPEM(cacertBytes); !ok {
+				t.Fatalf("Failed to parse %v", tc.trustBundleFile)
+			}
+
+			certzClient := gnsiC.Certz()
+			success := setupService.CertzRotate(t, cacert, certzClient, cert, san, serverAddr, testProfile, &serverCertEntity, &trustBundleEntity)
+			if !success {
+				t.Fatalf("%s:Certz rotation failed.", tc.desc)
+			}
+			t.Logf("%s:Certz rotation completed!", tc.desc)
+
+			// Replace config with newly added ssl profile after successful rotate.
+			servers = gnmi.GetAll(t, dut, gnmi.OC().System().GrpcServerAny().Name().State())
+			batch := gnmi.SetBatch{}
+			for _, server := range servers {
+				gnmi.BatchReplace(&batch, gnmi.OC().System().GrpcServer(server).CertificateId().Config(), testProfile)
+			}
+			batch.Set(t, dut)
+			t.Logf("%s:replaced gNMI config with new ssl profile successfully.", tc.desc)
+			time.Sleep(10 * time.Second)
```

Comment:
Can we add a comment that explains why we sleep here please?

Reply:
removed the sleep wait time.